### DISCUSSION TITLE
"npm run build" works on my machine, but not on the server

### DISCUSSION THREAD

**Discussion by rvyh** [vercel/next.js#25870] [Help]
"npm run build" works on my machine, a macbook with Node v12.18.3 and npm 6.14.8.

The server is running Ubuntu 20.04.1 LTS, node@v12.19.0 and npm@6.14.8.

The process gets stuck, this is the output:

```
info  - Loaded env from /root/project/.env.local
info  - Using webpack 5. Reason: no next.config.js https://nextjs.org/docs/messages/webpack5
info  - Checking validity of types  
info  - Creating an optimized production build .Killed
npm ERR! code ELIFECYCLE
npm ERR! errno 137
npm ERR! project@0.1.0 build: `next build`
npm ERR! Exit status 137
npm ERR! 
npm ERR! Failed at the project@0.1.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-06-07T23_03_23_256Z-debug.log
```

The log from the last line above doesn't include anything new.

The biggest change from last time it worked is a call to a MongoDB on Atlas (cloud) for which I store some constants at .env.local

As said locally on my machine everything works fine, "npm run dev" loads the project and "npm run build" throws no errors.

**Comment by kann4n** [comment]
same problem i have.
it works on my machine but not in vercel
